assets/src/blocks/edit.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Renders the edit interface for Meeting blocks
+ */
+import {InspectorControls, useBlockProps, BlockIcon} from '@wordpress/block-editor';
+import {
+	Panel,
+	PanelBody,
+	__experimentalNumberControl as NumberControl,
+	SelectControl,
+	Placeholder,
+	TextControl
+} from '@wordpress/components';
+import {__} from '@wordpress/i18n';
+
+/**
+ * @return {WPElement} Element to render.
+ * @param props
+ */
+const Edit = (props) => {
+	const {
+		attributes: {blocktype},
+		setAttributes
+	} = props;
+	const {serverSideRender: ServerSideRender} = wp;
+	return (
+		<div {...useBlockProps()}>
+			<InspectorControls>
+				<Panel>
+					<PanelBody title={__('Block Settings', '12-step-meeting-list')} icon="admin-plugins">
+						<SelectControl
+							label="Select Block Type"
+							value={props.attributes.blockType}
+							options={[
+								{label: 'Meetings UI', value: 'tsml_ui'},
+								{label: 'Next Meetings', value: 'tsml_next_meetings'},
+								{label: 'Types List', value: 'tsml_types_list'},
+								{label: 'Regions List', value: 'tsml_regions_list'}
```

Comment:
do these options show up anywhere? i don't see them in the block editor but i might be looking in the wrong place?

Reply:
Yes, if you add the "Meetings" block, and click on the block, that setting should show up as a dropdown.


<img width="1496" alt="Screenshot 2024-03-23 at 10 02 40 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/9162021/8c3af65d-9908-4b2a-beda-cdbe97864538">